src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/menu/AndroidShareMenu.java

```diff
@@ -0,0 +1,249 @@
+package io.github.thebusybiscuit.slimefun4.implementation.items.androids.menu;
+
+import io.github.thebusybiscuit.cscorelib2.inventory.ChestMenu;
+import io.github.thebusybiscuit.cscorelib2.item.CustomItem;
+import io.github.thebusybiscuit.cscorelib2.skull.SkullItem;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import io.github.thebusybiscuit.slimefun4.implementation.items.androids.ProgrammableAndroid;
+import io.github.thebusybiscuit.slimefun4.utils.ChatUtils;
+import io.github.thebusybiscuit.slimefun4.utils.ChestMenuUtils;
+import io.github.thebusybiscuit.slimefun4.utils.HeadTexture;
+import me.mrCookieSlime.Slimefun.api.BlockStorage;
+
+import org.apache.commons.lang.Validate;
+
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.Nonnull;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * The {@link AndroidShareMenu} is responsibility to handle trusted user using the {@link
+ * ProgrammableAndroid}
+ *
+ * @author StarWishsama
+ * @see ProgrammableAndroid
+ */
+public final class AndroidShareMenu {
+	private static final int startSlot = 9;
+	private static final int perPageDisplay = 45;
```

Comment:
here you say 45 per page, but when calculating the page count you use 36, which is correct? also maybe use SCREAMING_SNAKE_CASE because its a constant